pulsar/src/main/java/com/flipkart/varadhi/pulsar/entities/PulsarMessage.java

```diff
@@ -25,11 +29,41 @@ public PulsarOffset getOffset() {
         return new PulsarOffset(msg.getMessageId());
     }
 
+    @Override
```

Comment:
need to figure out, how to conform to the Message contracts around headers, message and group Id

Reply:
planning to take it independently ?